front-end/src/reducers/pages.js

```diff
@@ -1,9 +1,30 @@
 const defaultState = {
-
+  storiesList: {
+    searchResults: []
+  }
 }
 
 const pages = (state = defaultState, action) => {
-  return state;
+  switch (action.type) {
+    case 'STORIES_SEARCH_SUCCESS':
+      return {
+        ...state,
+        storiesList: {
+            ...state.storiesList,
+            searchResult: action.response
+        },
+      };
+    case 'CLEAR_SEARCH':
+    console.log('INSIDE CLEAR SEARCH');
+      return {
+        ...state,
+        storiesList: {
+          
```

Comment:
You're overwriting `storiesList` with the value `{}`. Reset the search results to an empty array.